starter-code/css/main.css

```diff
@@ -6,3 +6,172 @@ Green: #00B172
 White: #FFF
 
 */
+
+* {
+    margin: 0;
+    padding: 0;
+    font-family: sans-serif;
+}
+
+.nav-bar {
+    box-sizing: content-box;
+    display: flex;
+    justify-content: space-between;
+    background: #FFF;
+    position: fixed;
+    width: 100%;
+    align-items: center;
+    height: 60px;
+}
+
+.spotify-logo {
+    height: 50px;
+    width: auto;
+    float: left;
+    margin: 30px;
+}
```

Comment:
no float needed. You are using flex in the parent. For spacing you don't necessarily need margin all around the element. margin-left is enough. Even better: give the parent padding-left;)